mappings/net/minecraft/block/InfestedBlock.mapping

```diff
@@ -1,8 +1,11 @@
 CLASS bpe net/minecraft/block/InfestedBlock
 	FIELD a regularBlock Lbml;
-	FIELD b INFESTED_TO_REGULAR Ljava/util/Map;
+	FIELD b REGULAR_INFESTED_MAP Ljava/util/Map;
+	METHOD <init> (Lbml;Lbml$c;)V
+		ARG 1 regularBlock
+		ARG 2 settings
 	METHOD d getRegularBlock ()Lbml;
 	METHOD e getRegularBlock (Lbml;)Lbvj;
 		ARG 0 infestedBlockState
-	METHOD j hasRegularBlock (Lbvj;)Z
-		ARG 0 infestedBlockState
+	METHOD j isInfestableBlock (Lbvj;)Z
```

Comment:
I'd drop the *Block.

Reply:
`canInfest` is shorter.

Reply:
`canInfest` is not a good property, though. I'd go for `canBeInfested`, then. `isInfestable` is good, too.

Reply:
Can we just use `contains` given the class name `InfestableBlock` already has the `infest` context and this is a static method?

Reply:
No, `contains` would mean that the block somehow contains another block.